statedb_test.go

```diff
@@ -36,8 +44,15 @@ var (
 	BlockNumber4    = BlockNumber.Uint64() + 3
 	BlockHash5      = crypto.Keccak256Hash([]byte("I"))
 	BlockNumber5    = BlockNumber.Uint64() + 4
+	BlockHash6      = crypto.Keccak256Hash([]byte("am"))
+	BlockNumber6    = BlockNumber.Uint64() + 5
 	BlockParentHash = common.HexToHash("0123456701234567012345670123456701234567012345670123456701234567")
 
+	NonCanonicalHash4        = crypto.Keccak256Hash([]byte("I am a random non canonical hash"))
+	NonCanonicalBlockNumber4 = BlockNumber4
```

Comment:
I suggest we remove these number vars, to make it clear they're at the same height

Reply:
Good idea